<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QWidget" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1421</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>智联简历处理</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>1401</width>
     <height>651</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="fetch_result">
    <attribute name="title">
     <string>抓取结果</string>
    </attribute>
    <widget class="QTableWidget" name="fetch_table">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1261</width>
       <height>611</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>971</width>
       <height>0</height>
      </size>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="omo_integrate">
    <attribute name="title">
     <string>OMO集成</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>1280</x>
       <y>10</y>
       <width>101</width>
       <height>186</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vbox1">
      <property name="spacing">
       <number>50</number>
      </property>
      <item>
       <widget class="QPushButton" name="open_excel_button">
        <property name="text">
         <string>打开文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="update_omo_button">
        <property name="text">
         <string>更新到OMO</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="batch_download_button">
        <property name="text">
         <string>批量简历下载</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTableWidget" name="omo_table">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1261</width>
       <height>611</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>971</width>
       <height>0</height>
      </size>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="log">
    <attribute name="title">
     <string>日志</string>
    </attribute>
    <widget class="QTextEdit" name="log_textedit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1071</width>
       <height>571</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <!-- 选择校区 -->
  <widget class="QGroupBox" name="campus_group">
    <property name="geometry">
    <rect>
    <x>10</x>
    <y>10</y>
    <width>400</width>
    <height>120</height>
    </rect>
    </property>
    <property name="title">
    <string>选择校区</string>
    </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>371</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>80</number>
    </property>
    <item>
     <widget class="QPushButton" name="connect_zhilian_button">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #82b74b; color: white; border: none; padding: 8px 15px;</string>
      </property>
      <property name="text">
       <string>连接智联</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="fetch_info_button">
      <property name="styleSheet">
       <string notr="true">background-color: #82b74b; color: white; border: none; padding: 8px 15px;</string>
      </property>
      <property name="text">
       <string>批量打招呼并抓取信息</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
